test/extended/workspacetype/workspace.go

```diff
@@ -61,4 +62,67 @@ var _ = g.Describe("[sig-workspace]", func() {
 			return workSpaces
 		}, 60*time.Second, 5*time.Second).ShouldNot(o.ContainSubstring(myWorkSpace.Name))
 	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-I can list child workspaces under parent workspace [Smoke]", func() {
```

Comment:
Hi @Phaow, after some close inspections, I think that this case is a bit different because the `kubectl get` command in your case tests the existence of a workspace while in my case it test whether or not the `kubectl get` returns the full list of workspaces under a parent workspace. I suggest to keep this case.

Reply:
ok for me